NPE in MediaUtils.downloadExternalMedia

```
java.lang.RuntimeException: An error occured while executing doInBackground()
at android.os.AsyncTask$3.done(AsyncTask.java:299)
at java.util.concurrent.FutureTask$Sync.innerSetException(FutureTask.java:273)
at java.util.concurrent.FutureTask.setException(FutureTask.java:124)
at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:307)
at java.util.concurrent.FutureTask.run(FutureTask.java:137)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1076)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:569)
at java.lang.Thread.run(Thread.java:856)
Caused by: java.lang.NullPointerException
at android.content.ContentResolver.acquireExistingProvider(ContentResolver.java:1068)
at android.content.ContentResolver.getType(ContentResolver.java:216)
at org.wordpress.android.ui.media.MediaUtils.downloadExternalMedia(MediaUtils.java:347)
at org.wordpress.android.ui.posts.EditPostContentFragment$DownloadMediaTask.doInBackground(EditPostContentFragment.java:695)
at org.wordpress.android.ui.posts.EditPostContentFragment$DownloadMediaTask.doInBackground(EditPostContentFragment.java:690)
at android.os.AsyncTask$2.call(AsyncTask.java:287)
at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:305)
... 4 more
```

--- a/src/org/wordpress/android/ui/media/MediaUtils.java
+++ b/src/org/wordpress/android/ui/media/MediaUtils.java
@@ -343,7 +343,11 @@ public class MediaUtils {
     }
 
     public static Uri downloadExternalMedia(Context context, Uri imageUri) {
+        if (context == null || imageUri == null)
+            return null;
+
         File cacheDir;
+
         String mimeType = context.getContentResolver().getType(imageUri);
         boolean isVideo = (mimeType != null && mimeType.contains("video"));
 

